test/bemcontext-custom-error-logger-test.js

```diff
@@ -0,0 +1,43 @@
+var assert = require('assert');
+var bemhtml = require('../').bemhtml;
+
+describe('BEMContext: custom error logger', function() {
+  it('should use custom function from oninit', function() {
+    var templates = bemhtml.compile(function() {
+      oninit(function(exports) {
+        exports.BEMContext.prototype.onError = function(msg, e) {
+          console.info('>>> Error occurred', msg, e);
+        };
+      });
+
+      block('b1').attrs()(function() {
+        var attrs = applyNext();
+        attrs.foo = 'bar';
+        return attrs;
+      });
+    }, { production: true });
+
+    assert.doesNotThrow(function() {
+      assert.equal(templates.apply({ block: 'b1' }), '');
+    });
```

Comment:
@miripiruni I'm sorry, I can't see where do you check that this new redifined function is actually called?

Reply:
@vitkarpov omg, updated!